config/product/config.yaml

```diff
@@ -0,0 +1,47 @@
+# Generated output directory
+output_directory: ./output
+
+# Optional LLM to generate descriptions
+llm:
+  vendor: openai
+  model: gpt-3.5-turbo
+  # model: "gpt-4-turbo"
+  # model: "gpt-4o"
```

Comment:
as I mentioned in the slack message, I recommend doing prompt-engineering and/or swapping to a better model for generating example descriptions, to avoid weird stuff showing  up in descriptions.